### PR TITLE
Modify marshmallow.md and add a new path to the story to Trip-Yoursel…

### PR DESCRIPTION
**Pull request by manas94gupta**
…f-Hue

### CODE REVIEW

**Comment by manas94gupta** [comment]
@casey-collab, add my path to the story

**Comment by casey-collab** [comment]
It looks like this pull request deletes quite a few lines. Was this a mistake?

**Comment by manas94gupta** [comment]
@casey-collab, add my path to the story.

File: english/Trip-Yourself-Hue/trippy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Choose your poison, get your brains all mooshed up -
+--> [Pony](https://www.youtube.com/watch?v=X0gyqsPdWFE)
+--> [Horse](https://www.youtube.com/watch?v=QCGcCC065CY)
+--> [Cake](https://vimeo.com/128877443)
```
Comment (casey-collab) [inline english/Trip-Yourself-Hue/trippy.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by manas94gupta** [comment]
@casey-collab, Add!

File: english/Trip-Yourself-Hue/trippy.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Choose your poison, get your brains all mooshed up    
+---> [Pony](https://www.youtube.com/watch?v=X0gyqsPdWFE)   
+--> [Horse](https://www.youtube.com/watch?v=QCGcCC065CY)
```
Comment (casey-collab) [inline english/Trip-Yourself-Hue/trippy.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/Trip-Yourself-Hue/trippy.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Choose your poison, get your brains all mooshed up    
+---> [Pony](https://www.youtube.com/watch?v=X0gyqsPdWFE)   
+--> [Horse](https://www.youtube.com/watch?v=QCGcCC065CY)   
+--> [Cake](https://vimeo.com/128877443)
```
Comment (casey-collab) [inline english/Trip-Yourself-Hue/trippy.md]: This link is also on the line directly after another link.

File: english/Trip-Yourself-Hue/trippy.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Choose your poison, get your brains all mooshed up    
+---> [Pony](https://www.youtube.com/watch?v=X0gyqsPdWFE)   
+--> [Horse](https://www.youtube.com/watch?v=QCGcCC065CY)   
+--> [Cake](https://vimeo.com/128877443)   
+--> [MaryJane](https://www.youtube.com/watch?v=JIpv81BH8Wo)
```
Comment (casey-collab) [inline english/Trip-Yourself-Hue/trippy.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by manas94gupta** [comment]
@casey-collab, Add!

**Comment by casey-collab** [comment]
Looks great! :seedling:

**Pull request merged**